### ISSUE TITLE
Failed to retrieve Uniswap trades

### ISSUE THREAD

**Issue by leontodd** [rotki/rotki#1896]
## Problem Definition

I bought premium and began to set everything up. I imported my uniswap liquidity pools just fine at first. After that I could not import anything from Uniswap, both pools and swaps. The error attached below appears. I have tried clearing the cache, removing addresses and re-adding them, and clearing all Rotki data from AppData and restarting the app.

## Logs

```
Failed to retrieve Uniswap trades: Failed to query the graph for query ($limit: Int!, $offset: Int!, $address: Bytes!, $start_ts: BigInt!, $end_ts: BigInt!){swaps ( first: $limit, skip: $offset, where: { to: $address, timestamp_gte: $start_ts, timestamp_lte: $end_ts, } ) { transaction { swaps { id logIndex sender to timestamp pair { token0 { id decimals name symbol } token1 { id decimals name symbol } } amount0In amount0Out amount1In amount1Out } } }} due to Field "swaps" argument "subgraphError" of type "_SubgraphErrorPolicy_!" is required but not provided.
```

### System Description

Operating system: Windows 10 Pro
Rotki version: 1.9.1

**Comment by LefterisJP** [comment]
Hello @leontodd. This is a known issue and it's related to the graph protocol that we are querying for historical data for premium users.

They have unfortunately pushed an update that breaks all python implementations when querying the graph. We are working with their engineers so that they provide a fix on their side as soon as possible. Their earliest estimation was on Monday.

Early next week, also Monday hopefully we will also push a release on our side that handles the bug they have but on our side. The release is also adding a bunch of new features.

So for now you can:

1. Wait until Monday and the new release.
2. Try out our current development version which already includes the fix. But you would need to be able to build by source.
3.Remove the premium key from rotki temporarily (this way graph is no longer queried)

**Comment by leontodd** [comment]
That's great, I will mark this as closed as the fix already merged. Thank you for the help!

**Comment by LefterisJP** [comment]
Hey @leontodd. More premium users said they were affected by this and I felt we should act so I decided to make an 1 line patch release with the fix on our side. Can't wait for the graph guys to act until Monday.

So v1.9.2 is now out. It just adds this 1 line fix on top of v1.9.1 and should fix your issue: https://github.com/rotki/rotki/releases/tag/v1.9.2

**Comment by LefterisJP** [comment]
Keep in mind that v1.10.0 is also coming early next week :smile:
